### ISSUE TITLE
PIxel type compatibility

### ISSUE THREAD

**Issue by NicoKiaru** [BIOP/abba_python#25]
Check Brainglobe atlas kim_dev_mouse_idisco

See https://forum.image.sc/t/mri-template-for-ccfv3-space/101219/7

**Comment by adamltyson** [comment]
How does the original Allen atlas work in ABBA? That has lots of these v. high integers right?

I agree there appears no need to use these values, but they seem to have been decided by the Allen (I presume for good reasons I'm not aware of). We debated changing these within BrainGlobe, but that seemed a step too far towards standardisation.

**Comment by NicoKiaru** [comment]
> How does the original Allen atlas work in ABBA?

I cheated because at the time I did not know how to circumvent the 16 bits unsigned int limitation. So I looked for a way and found that there was still unicity of the labels if you modulo them by 65000. So I re-indexed the labels to labels mod 65000 (https://zenodo.org/records/7492551).

At the time I did not think of supporting other atlas and this trick worked for a while. I guess it's time to move on.
